src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Welcome'
+heading: 'xk6-disruptor Documentation'
+excerpt: 'xk6-disruptor is a k6 extension providing fault injection capabilities to test system reliability under turbulent conditions. Think of it as unit testing, but for reliability.'
+---
+
+xk6-disruptor is a k6 extension that can inject faults into a system to simulate turbulent conditions. 
+
+xk6-browser intends to help testers approach the discipline of "Chaos Engineering" with k6 way&mdash;providing the best developer experience as a primary objective. 
```

Comment:
Fixing the first one

Regarding the "best developer experience" I agree but I think it can be done in a follow-up PR where we revise the whole project position.